starter_code/app.js

```diff
@@ -1,21 +1,41 @@
 
 const express = require('express');
-const hbs     = require('hbs');
-const app     = express();
 const path    = require('path');
+const expressLayouts = require('express-ejs-layouts');
 const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 const punkAPI = new PunkAPIWrapper();
```

Comment:
Try to separate the requires from other variables that you add. It's a way to keep your code more structure.

Reply:
Noted!